tensorflow/contrib/tensorrt/__init__.py

```diff
@@ -18,10 +18,12 @@
 from __future__ import division
 from __future__ import print_function
 
+from tensorflow.python.framework import errors
+
 # pylint: disable=unused-import,wildcard-import,g-import-not-at-top
 try:
   from tensorflow.contrib.tensorrt.python import *
-except Exception as e:
+except errors.NotFoundError as e:
```

Comment:
I cannot see any code path that raises this specific error message.
From what I can see this is the error you will see:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/framework/load_library.py#L55

Reply:
It's raised from [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/framework/errors_impl.py#L516).